test/Microsoft.ML.Tests/Scenarios/BinaryClassification.cs

```diff
@@ -0,0 +1,92 @@
+﻿using Microsoft.ML.Data;
+using Microsoft.ML.Models;
+using Microsoft.ML.Runtime.Api;
+using Microsoft.ML.Runtime.Data;
+using Microsoft.ML.Trainers;
+using System.IO;
+using Xunit;
+
+namespace Microsoft.ML.Scenarios
+{
+    public partial class ScenariosTests
+    {
+        public class BreastCancerData
+        {
+            [Column(ordinal: "0")]
```

Comment:
>ordinal: "0 [](start = 20, length = 11)

OK. As near as I can see we're not using the text loader wizard. So what is this for?

Reply:
Old habits.

---
In reply to: [191497078](https://github.com/dotnet/machinelearning/pull/248#discussion_r191497078) [](ancestors = 191497078)